docs/content/en/docs-v0.47.x/installation/install-control-plane/installing-controlplane-on-k8s.md

```diff
@@ -72,9 +74,9 @@ helm upgrade -i pipecd oci://ghcr.io/pipe-cd/chart/pipecd --version {{< blocks/l
   --set-file secret.gcsServiceAccount.data=path-to-service-account-file
 ```
 
-Currently, besides `Firestore` PipeCD supports other databases as its datastore such as `MySQL`. Also as for filestore, PipeCD supports `AWS S3` and `MINIO` either.
+#### Using MySQL and MINIO
 
-For example, in case of using `MySQL` as datastore and `MINIO` as filestore, the ControlPlane configuration will be as follow:
+For example, in case of using `MySQL` as datastore and `MINIO` as filestore, remember that url of database should be in format: `<user>:<password>@tcp(<ip>:<port>)`, the ControlPlane configuration will be as follow:
```

Comment:
[nits]
> remember that url of database should be in format: `<user>:<password>@tcp(<ip>:<port>)`

How about moving it to the following yaml to clarify where this sentence mentions? 
like this:

```yaml
  datastore:
    type: MYSQL
    config:
      url: {YOUR_MYSQL_ADDRESS} # format: <user>:<password>@tcp(<ip>:<port>)

```